deviceModeInit.js

```diff
@@ -212,6 +240,11 @@ var rudderTracking = (function () {
     });
   }
 
+  // function to format value to fload to to fixed places
+  function formatPrice(value) {
+    return parseFloat(value / 100).toFixed(2);
```

Comment:
String also works, but if its "abc" it would return NaN. For these fields I have verified these are numeric there wont be any unhanded exceptions here

Reply:
Ok. I'd throw in some error handling here, just to be safe.